schema/component.schema.json

```diff
@@ -0,0 +1,210 @@
+              "type": "boolean",
+              "title": "Show related videos",
+              "description": "Since Sept 2018 the YouTube player no longer allows you to prevent related videos from being shown - see https://developers.google.com/youtube/player_parameters#rel",
+              "default": false
+            },
+            "_loop": {
+              "type": "boolean",
+              "title": "Loop",
+              "description": "Whether to play the video on a loop or not.",
+              "default": false
+            },
+            "_modestBranding": {
+              "type": "boolean",
+              "title": "Modest branding",
+              "description": "This parameter lets you use a YouTube player that does not show a YouTube logo. Set to true to prevent the YouTube logo from displaying in the control bar. Note that a small YouTube text label will still display in the upper-right corner of a paused video when the user's mouse pointer hovers over the player.",
+              "default": true
+            },
+            "_playbackQuality": {
+              "type": "string",
+              "title": "Playback quality",
+              "description": "The suggested video quality for the video. You should select a playback quality that corresponds to the size of your video player. For example, if your page displays a 1280px by 720px video player, a hd720 quality video will actually look better than an hd1080 quality video.",
+              "default": "default",
+              "enum": [
+                "default",
+                "small",
+                "medium",
+                "large",
+                "hd720",
+                "hd1080",
+                "highres"
+              ],
+              "_backboneForms": "Select"
+            },
+            "_showAnnotations": {
+              "type": "boolean",
+              "title": "Video annotations",
+              "description": "Whether video annotations should be shown or not.",
+              "default": false
+            },
+            "_progressColor": {
+              "type": "string",
+              "title": "Progress bar color",
+              "description": "The colour that will be used in the player's video progress bar to highlight the amount of the video that the viewer has already seen. Setting this to 'white' will disable the 'modest branding' option.",
+              "default": "default",
+              "enum": [
+                "red",
+                "white"
+              ],
+              "_backboneForms": "Select"
+            }
+          }
+        },
+        "_transcript": {
+          "type": "object",
+          "title": "Transcript Options",
+          "default": {},
+          "properties": {
+            "_setCompletionOnView": {
+              "type": "boolean",
+              "title": "Trigger completion?",
+              "description": "Whether to have this component mark as completed when the transcript is shown.",
```

Comment:
Should the language we use in this and the following object title / descriptions be synced with the text in the media plugin / should we update the media versions with this text?

Reply:
Carried transcript language across media, vimeo & youtube components.